drivers/hdlc_rcp_if/Kconfig

```diff
@@ -0,0 +1,13 @@
+# HDLC RCP communication Interface configuration options
+
+# Copyright (c) 2024 NXP
+# SPDX-License-Identifier: Apache-2.0
+
+#
+# Openthead HDLC communication Interface for a Zephyr RCP host
+#
+
+menuconfig OPENTHREAD_HDLC_RCP
+	bool "OPENTHREAD HDLC interface for a Zephyr RCP host"
+	depends on NET_L2_OPENTHREAD
```

Comment:
I also wonder, this should probably be prohibited if someone build OT coprocessor (`depends on !OPENTHREAD_COPROCESSOR`), those two build types seems to be in conflict?

Reply:
resolved